jsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "./.svelte-kit/tsconfig.json",
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
```

Comment:
Take this option out, it's just going to cause a lot of type checking where it isn't needed. 

The rest looks good, happy to merge after the change.

Reply:
Fixed. I had meant to not have this set, but must have accidentally got into the commit I pushed.